tests/cpgrid/adapt_cpgrid_test.cpp

```diff
@@ -208,8 +208,8 @@ void markAndAdapt_check(Dune::CpGrid& coarse_grid,
         const auto& grid_view = coarse_grid.leafGridView();
 
         Dune::MultipleCodimMultipleGeomTypeMapper<Dune::CpGrid::LeafGridView> adaptMapper(grid_view, Dune::mcmgElementLayout());
-        const auto& adapt_idSet = adapted_leaf.local_id_set_;
-
+        const auto& leaf_localIdSet = data.back()->local_id_set_;
```

Comment:
Maybe get the id set from the grid.
```suggestion
        const auto& leaf_localIdSet = grid_view.grid().localIdSet();
```

Reply:
Similar as before, I use the CpGridData ones, and grid.globalIdSet() sometimes too